composer.json

```diff
@@ -54,7 +54,8 @@
         "netgen/layouts-ibexa": "~1.4.0",
         "netgen/layouts-ibexa-site-api": "~1.4.0",
         "netgen/layouts-ibexa-relation-list-query": "^1.4",
-        "netgen/layouts-ibexa-tags-query": "^1.4"
+        "netgen/layouts-ibexa-tags-query": "^1.4",
+        "pentatrion/vite-bundle": "^6.4"
```

Comment:
Can you please move this just below `"tanoconsulting/ibexa-migration-bundle", to keep extra dependencies not related to Netgen together?